sklearn/setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import find_packages
```

Comment:
Is this file being used?
I think we're also trying to move away from this format of a setup.py file, but let me double check.